core/domain/user_services.py

```diff
@@ -495,6 +495,96 @@ def get_user_settings_by_auth_id(
         return None
 
 
+def get_all_user_groups() -> List[Dict[str, Union[str, List[str]]]]:
+    """Return the list of user groups dictionary data having key as user-group
+    name and value as list of users associated to the user-group.
+
+    Returns:
+        user_group_models_dict: List[Dict[str, Union[str, List[str]]]]. List of
```

Comment:
Return value shouldn't be named in the function docstring.

Reply:
Got it, I have updated this.